utils.go

```diff
@@ -340,6 +340,11 @@ func setBridgeIP(bridgeName, bridgeIP, bridgeSubnet string) error {
 }
 
 func setupBridge(n *NetConf) (*netlink.Bridge, error) {
+	// bridge exists while using some networking models.
+	if n.SkipBridgeConfigureIP {
```

Comment:
Shouldn't this be below at https://github.com/rancher/rancher-cni-bridge/pull/18/files#diff-7d1c2a3334601b6c1958aae0a594cba8R354